src/eval.rs

```diff
@@ -353,3 +345,76 @@ pub fn eval_entry<'tcx>(
         Err(e) => report_error(&ecx, e),
     }
 }
+
+// Turns an array of arguments into a Windows command line string.
+//
+// The string will be UTF-16 encoded and NUL terminated.
+// 
+// Panics if the zeroth argument contains the `"` character because doublequotes
+// in argv[0] cannot be encoded using the standard command line parsing rules.
```

Comment:
```suggestion
/// Turns an array of arguments into a Windows command line string.
///
/// The string will be UTF-16 encoded and NUL terminated.
/// 
/// Panics if the zeroth argument contains the `"` character because doublequotes
/// in argv[0] cannot be encoded using the standard command line parsing rules.
```

Private functions are visible in the documentation when using `--document-private-items`.

Reply:
Ah thanks! Should be fixed now (along with the formatting).